aggregate_test.go

```diff
@@ -328,9 +329,10 @@ func Test_Aggregate(t *testing.T) {
 			ErrorMatcher:           nil,
 		},
 
-		// Test 5 ensures when having a operator capability containing two bundles
-		// [a1,a2] and having another operator capability containing one bundle
-		// [b1], there should be two aggregated capabilities [[a1,b1],[a2,b1]].
+		// Test 5 ensures when having an operator's list of version bundles
```

Comment:
Same here, maybe I am just a bit tired but the descriptions for Test 4,5&6 just run on in a single sentence.

Reply:
Done.